### PR TITLE
Content State Update API changes to sync course assessment events to kafka

### PR DESCRIPTION
**Pull request by sowmya-dixit**

### CODE REVIEW

File: actors/sunbird-lms-mw/actors/common/src/main/java/org/sunbird/learner/actors/LearnerStateUpdateActor.java [hunk 1]
```diff
@@ -400,4 +416,17 @@ private void pushInstructionEvent(
     String topic = ProjectUtil.getConfigValue("kafka_topics_instruction");
     InstructionEventGenerator.pushInstructionEvent(topic, data);
   }
+
+  private void syncAssessmentData(Map<String, Object> assessmentData) throws Exception {
+    ObjectMapper mapper = new ObjectMapper();
```
Comment (maheshkumargangula) [inline actors/sunbird-lms-mw/actors/common/src/main/java/org/sunbird/learner/actors/LearnerStateUpdateActor.java]: Create ObjectMapper as Global variable.

File: actors/sunbird-lms-mw/actors/common/src/main/java/org/sunbird/learner/actors/LearnerStateUpdateActor.java [hunk 2]
```diff
@@ -69,6 +72,19 @@ public void onReceive(Request request) throws Throwable {
 
     if (request.getOperation().equalsIgnoreCase(ActorOperations.ADD_CONTENT.getValue())) {
       String userId = (String) request.getRequest().get(JsonKey.USER_ID);
+      List<Map<String, Object>> assessments =
+              (List<Map<String, Object>>) request.getRequest().get(JsonKey.ASSESSMENT_EVENTS);
+      if (null != assessments && !assessments.isEmpty()) {
```
Comment (maheshkumargangula) [inline actors/sunbird-lms-mw/actors/common/src/main/java/org/sunbird/learner/actors/LearnerStateUpdateActor.java]: Use `CollectionUtils.isEmpty(assessments).

Comment (sowmya-dixit) [reply actors/sunbird-lms-mw/actors/common/src/main/java/org/sunbird/learner/actors/LearnerStateUpdateActor.java]: Added empty checks using CollectionUtils

File: actors/sunbird-lms-mw/actors/common/src/main/java/org/sunbird/learner/actors/LearnerStateUpdateActor.java (hunk 1 above)
Comment (sowmya-dixit) [reply actors/sunbird-lms-mw/actors/common/src/main/java/org/sunbird/learner/actors/LearnerStateUpdateActor.java]: Created ObjectMapper as Global variable

**Pull request merged**